pkg/goldpinger/updater.go

```diff
@@ -153,6 +153,17 @@ func updateCounters() {
 		}
 	}
 	CountHealthyUnhealthyNodes(counterHealthy, float64(len(checkResults.PodResults))-counterHealthy)
+	// check DNS, but don't block the access to checkResultsMux
+	go func(healthy bool) {
+		if healthy {
+			for _, response := range *checkDNS() {
+				if response.Error != "" {
+					healthy = false
+				}
+			}
+		}
+		SetClusterHealth(healthy)
+	}(int(counterHealthy) == len(checkResults.PodResults))
```

Comment:
This is super duper nitpicky but can we create a var like: `nodesHealthy := int(counterHealthy) == len(checkResults.PodResults)` before the function and then pass it in to the function. Sorry I am super slow, just took me a couple of reads before I got the `ah ha, of course that makes sense`...